www/client/containers/DataVisualization/ComponentPage.js

```diff
@@ -0,0 +1,77 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Media from 'react-media';
+import { Spinner } from '@momentum-ui/react';
+import PageHeader from '../../momentum-ui/PageHeader';
+import CodePen from '../../components/CodePen';
+
+const chartsExamples = require('../../generated/examples/charts/chartsList');
+const addScript = () => {
+  if(!document.querySelector('SCRIPT[codepen]')){
+    let _script = document.createElement('SCRIPT');
+    _script.src= 'https://static.codepen.io/assets/embed/ei.js';
+    _script.setAttribute('codepen','codepen');
+    document.getElementsByTagName('HEAD')[0].appendChild(_script);
+  }
+};
+class ComponentPage extends React.Component {
```

Comment:
Should this also be considered ComponentPage or DataVisPage?

Reply:
CodePen API does not have NPM package version.